packages/manager/apps/pci-kubernetes/src/helpers/index.ts

```diff
@@ -203,3 +203,8 @@ export function generateUniqueName(
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+export const isMonoZone = (type: DeploymentMode) =>
+  type === DeploymentMode.MONO_ZONE;
+export const isMultiZones = (type: DeploymentMode) =>
+  type === DeploymentMode.MULTI_ZONES;
```

Comment:
As it's an helper, we could enhance a bit naming to be more accurate regarding usage:
```suggestion
export const isMonoDeploymentZone = (type: DeploymentMode) =>
  type === DeploymentMode.MONO_ZONE;
export const isMultiDeploymentZones = (type: DeploymentMode) =>
  type === DeploymentMode.MULTI_ZONES;
```